test/unit/node/routes/login.test.ts

```diff
@@ -92,5 +92,38 @@ describe("login", () => {
 
       expect(htmlContent).toContain("Incorrect password")
     })
+
+    it("should return correct app-name", async () => {
+      process.env.PASSWORD = previousEnvPassword
+      const appName = "testnäme"
+      const codeServer = await integration.setup([`--app-name=${appName}`], "")
+      const resp = await codeServer.fetch("/login", { method: "GET" })
+
+      const htmlContent = await resp.text()
+      expect(resp.status).toBe(200)
+      expect(htmlContent).toContain(appName)
+    })
+
+    it("should return correct welcome text", async () => {
+      process.env.PASSWORD = previousEnvPassword
+      const welcomeText = "Welcome to your code workspace! öäü🔐"
```

Comment:
nit: should we use characters without accent marks in case they're rendered differently on someone's machine?
```suggestion
      const welcomeText = "Welcome to your code workspace!"
```

Reply:
my idea was to test if there are any implications when using unicode, but I don't know if that should be there

Reply:
I think that could be tested separately if needed but I personally don't think we need to. thoughts @code-asher ?

Reply:
yeah I don't think it will do any harm, should be ready to merge. I would be glad if you could add the hacktoberfest-accepted label so it counts for https://hacktoberfest.com :)

Reply:
Seems chill to me :+1: